projects/web/src/app/meetups-list/meetups-list.component.scss

```diff
@@ -0,0 +1,86 @@
+@import '../../theme';
+@import '../../mixins';
+@import '../../variables';
```

Comment:
¿Qué tal si aplicamos esto al proyecto.?
Podría ser importar todo desde styles.

Quedaría algo así como:
`@import 'styles/theme'`
`@import 'styles/mixins'`
`@import 'styles/variables'`

https://scotch.io/tutorials/angular-shortcut-to-importing-styles-files-in-components#toc-shortcut-with-angular-cli-configuration

Reply:
Me gusta la idea, pero...creo que podría ser otra tarea a partir de esta.